common/src/main/java/com/microsoft/identity/common/internal/fido/FidoRequestField.kt

```diff
@@ -25,11 +25,13 @@ package com.microsoft.identity.common.internal.fido
 /**
  * Represents FIDO request fields shared among registration and authentication requests.
  */
-enum class FidoRequestField {
-    Challenge,
-    RelyingPartyIdentifier,
-    UserVerificationPolicy,
-    Version,
-    SubmitUrl,
-    Context
+class FidoRequestField {
```

Comment:
Changing class type, since enums are capitalized by convention. (Had to change the values to match what the server side implemented for the protocol)

Reply:
Changes made here: 2112dd72efe46ae748ab0b2525b21810e46df28a